MapboxSearch/base/src/main/java/com/mapbox/search/base/location/LocationEngineAdapter.kt

```diff
@@ -54,11 +67,21 @@ class LocationEngineAdapter(
 
     private fun startLocationListener() {
         locationProvider?.addLocationObserver(locationObserver)
+
+        locationEngineObservationTimeout?.let { timeout ->
```

Comment:
Could you please add more details why do you need this? Just to remove location observer?

Reply:
Here is the full story:
https://mapbox.atlassian.net/browse/SSDK-552